from core.exceptions import BaseError


class TelegramAuthError(BaseError): ...
